src/Security/Member.php

```diff
@@ -382,7 +382,7 @@ public function canLogIn()
      *
      * @return bool
      */
-    public function isLockedOut()
+    protected function isLockedOut()
```

Comment:
I've got mixed feelings about this change — I agree that exposing some clearer public APIs that developers can rely on is a good idea (i.e. recommend `canLogin()` over `isLockedOut()`). However, I think that it's a bit more work than simply marking this as protected and so I don't see the change of visibility here as being of a heck of a lot of value without further work.